### ISSUE TITLE
[BUG] print(DataFrame) doesn't present its data in terminal in eager mode 

### ISSUE THREAD

**Issue by wdkwyf** [mars-project/mars#508]
**Describe the bug**
In eager mode, print a DataFrame doesn't output it's data.

**To Reproduce**
1. Your Python version
3.7.3
2. The version of Mars you use
v0.2.0b1
3. Versions of crucial packages, such as numpy, scipy and protobuf
DataFrame
4. Full stack of the error.
Unwanted behavior:
print(DataFrame) in eager mode doesn't print the data of the DataFrame.
5. Minimized code to reproduce the error.
When I tried:
```
import mars.dataframe as md
from mars.config import options
options.eager_mode = True

a = md.DataFrame({'angles': [0, 3, 4]})
print(a)
```
The output is:
`<mars.dataframe.core.DataFrameData object at 0x7fafe8e54208>
`
which is not right.
**Expected behavior**
Output should be:
```
   angles
0       0
1       3
2       4
```

**Comment by wjsi** [comment]
This may have something to do with #490 which will be fixed soon.

**Comment by sighingnow** [comment]
I have opened #511 to catch the release timeline. Sorry for that I didn't notice the assignees field of this issue.

**Comment by wdkwyf** [comment]
> I have opened #511 to catch the release timeline. Sorry for that I didn't notice the assignees field of this issue.

No problem. I haven't started doing this yet.
Thanks for your contributions.

**Comment by qinxuye** [comment]
> This may have something to do with #490 which will be fixed soon.

This issue is due to the `repr` of DataFrame that the data is not shown when eager mode is on. It's not strongly related to corner or not. They can be fixed separately.

**Comment by NJUPole** [comment]
> This issue is due to the `repr` of DataFrame that the data is not shown when eager mode is on. It's not strongly related to corner or not. They can be fixed separately.

Agree, we can merge #511 firstly to solve this issue.

**Comment by sighingnow** [comment]
Fixed by #511.
